src/pygeosx/pygeosx.cpp

```diff
@@ -383,5 +408,9 @@ PyInit_pygeosx()
   }
 
   // Since we return module we don't want to decrease the reference count.
-  return module.release();
+  return module;
```

Comment:
Why this change?

Reply:
Good question... That actually could explain those PyModule cleaning error at the end of my simulations !